src/main/java/org/mockito/internal/listeners/StubbingLookupNotifier.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2018 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.internal.listeners;
+
+import org.mockito.internal.creation.settings.CreationSettings;
+import org.mockito.invocation.Invocation;
+import org.mockito.mock.MockCreationSettings;
+import org.mockito.stubbing.Stubbing;
+
+import java.util.Collection;
+import java.util.List;
+
+public class StubbingLookupNotifier {
+
+    public static void notifyStubbedAnswerLookup(Invocation invocation, Stubbing stubbingFound,
+                                                 Collection<Stubbing> allStubbings, CreationSettings creationSettings) {
+        List<StubbingLookupListener> listeners = creationSettings.getStubbingLookupListeners();
+        if (listeners.isEmpty()) {
+            return;
+        }
+        // TODO #793 - onStubbingLookup() should not be called if stubbingFound is null.
```

Comment:
> onStubbingLookup() should not be called if stubbingFound is null.

Why? We actually have code that needs to act when stubbing is not found (e.g. strictness - detecting argument mismatch).

Reply:
Oh, I misunderstood that this listener should be called only if the stubbed answer is found.
I will remove this comment!